webapp/src/components/table/table.tsx

```diff
@@ -26,6 +29,39 @@ type Props = {
     onCardClicked: (e: React.MouseEvent, card: Card) => void
 }
 
+// Eventually, these will need to come from the theme.
+// Currently, the theme doesn't change any padding or font descriptor
+const header = {
+    fontDescriptor: 'bolder 14px sans-serif',
+    padding: 22,
+}
+const select = {
+    fontDescriptor: 'bolder 13px sans-serif',
+    padding: 56,
+}
+const text = {
+    fontDescriptor: '14px sans-serif',
+    padding: 30,
+}
+const title = {
+    fontDescriptor: '14px sans-serif',
+    padding: 57,
+}
+
+// re-use canvas object for better performance
+const canvas = document.createElement('canvas') as HTMLCanvasElement
```

Comment:
Minor: might want to move this to a shared utility class later. I can see this being useful for lots of things.

Reply:
Yeah, I debated on whether to do it now.

Reply:
Moved it to Utils.